cypress/integration/show_property.spec.js

```diff
@@ -70,44 +134,30 @@ describe('Show property', () => {
             cy.contains('th', 'Description')
           })
```

Comment:
Should we also expect the existing properties to be shown in this test? At the moment it could seem like this replaces the existing properties but I think our requirement is to add more to an existing list. If this test is getting long as a result, we could split out a separate test, e.g. `cypress/integration/property_show/repair_history_tab.spec.js` or similar